client/logmon/client.go

```diff
@@ -24,12 +27,18 @@ func (c *logmonClient) Start(cfg *LogConfig) error {
 		StdoutFifo:     cfg.StdoutFifo,
 		StderrFifo:     cfg.StderrFifo,
 	}
-	_, err := c.client.Start(context.Background(), req)
+	ctx, cancel := context.WithTimeout(context.Background(), logmonRPCTimeout)
+	defer cancel()
+
+	_, err := c.client.Start(ctx, req)
```

Comment:
I believe Start here returns quickly, but logmon blocks in background until task is opened (e.g. docker image is downloaded and run), so setting a relatively short timeout should be fine.